website/pages/en/index.js

```diff
@@ -103,22 +103,50 @@ class Index extends React.Component {
       </Block>
     );
 
-    const Disclaimer = () => (
```

Comment:
We should remove this from our main README as well

Reply:
Done.